pontoon/machinery/automl_warmup.py

```diff
@@ -0,0 +1,47 @@
+"""
+Google Cloud AutoML Translation has latency of ~15s, caused by the loading time of a
+custom model into the chip. To keep latency low, we need to make regular dummy warm-up
```

Comment:
```suggestion
custom model into the chip. To keep latency low, we need to make regular dummy warmup
```

That's how you spell it in the rest of the files. Personally, I prefer the hyphenated version (`warm-up`).